index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+  <title>My Portfolio</title>
+</head>
+<body>
+  <header class="header">
+    <div class="header2">
+      <div class="logo_welcome">
+        <a href="#" class="wlc">Welcome</a>
+      </div>
+        <a href="#" class ="union"><svg width="18" height="12" viewBox="0 0 18 12" fill="none" xmlns="http://www.w3.org/2000/svg">
+          <path fill-rule="evenodd" clip-rule="evenodd" d="M0.00012207 1C0.00012207 0.447715 0.447837 0 1.00012 0H17.0001C17.5524 0 18.0001 0.447715 18.0001 1C18.0001 1.55228 17.5524 2 17.0001 2H1.00012C0.447837 2 0.00012207 1.55228 0.00012207 1ZM0.00012207 6C0.00012207 5.44772 0.447837 5 1.00012 5L17.0001 5C17.5524 5 18.0001 5.44772 18.0001 6C18.0001 6.55229 17.5524 7 17.0001 7L1.00012 7C0.447837 7 0.00012207 6.55228 0.00012207 6ZM1.00012 10C0.447837 10 0.00012207 10.4477 0.00012207 11C0.00012207 11.5523 0.447837 12 1.00012 12H17.0001C17.5524 12 18.0001 11.5523 18.0001 11C18.0001 10.4477 17.5524 10 17.0001 10H1.00012Z" fill="#28352F"/>
+          </svg></a>
```

Comment:
:red_circle: As requested by the previous reviewer Kindly make sure your hamburger menu is wrapped in a nav tag. This code should be surrounded by `<nav>.......</nav>` as follow :arrow_down: 

```
<nav>
  <a href="#" class ="union"><svg width="18" height="12" viewBox="0 0 18 12" fill="none" xmlns="http://www.w3.org/2000/svg">
          <path fill-rule="evenodd" clip-rule="evenodd" d="M0.00012207 1C0.00012207 0.447715 0.447837 0 1.00012 0H17.0001C17.5524 0 18.0001 0.447715 18.0001 1C18.0001 1.55228 17.5524 2 17.0001 2H1.00012C0.447837 2 0.00012207 1.55228 0.00012207 1ZM0.00012207 6C0.00012207 5.44772 0.447837 5 1.00012 5L17.0001 5C17.5524 5 18.0001 5.44772 18.0001 6C18.0001 6.55229 17.5524 7 17.0001 7L1.00012 7C0.447837 7 0.00012207 6.55228 0.00012207 6ZM1.00012 10C0.447837 10 0.00012207 10.4477 0.00012207 11C0.00012207 11.5523 0.447837 12 1.00012 12H17.0001C17.5524 12 18.0001 11.5523 18.0001 11C18.0001 10.4477 17.5524 10 17.0001 10H1.00012Z" fill="#28352F"/>
          </svg></a>
</nav>
```

The `nav` tag is for semantics reason, and the right position for convention and user experience for mobile devices.